api/models/vpp/abf/abf.proto

```diff
@@ -0,0 +1,32 @@
+syntax = "proto3";
+
+package vpp.abf;
+
+option go_package = "github.com/ligato/vpp-agent/api/models/vpp/abf;vpp_abf";
+
+import "github.com/gogo/protobuf/gogoproto/gogo.proto";
+option (gogoproto.messagename_all) = true;
+
+// ACL based forwarding
+message ABF {
+    string index = 1;               // ABF index (must be a number)
```

Comment:
If it must be a number why define it as string type?

Reply:
Good question. I wanted to keep it in line with similar models which require numerical identifier and do not support tags (like IPSec). Before it was more convenient to do it as such. 
I'll change it here, and we should do it likewise in other similar models to have consistent API.